core/src/main/java/nl/nn/adapterframework/configuration/Configuration.java

```diff
@@ -522,7 +524,18 @@ public ConfigurationWarnings getConfigurationWarnings() {
 		return null;
 	}
 
+	// Dummy setter to allow SapSystems being added to Configurations via Frank!Config XSD
+	public void setSapSystems(SapSystems sapSystems) {
+		// SapSystems self register;
+	}
+
+	// Dummy setter to allow JmsRealms being added to Configurations via Frank!Config XSD
+	public void setJmsRealms(JmsRealms realm) {
+		// JmsRealm-objects self register in JmsRealms-class;
+	}
+
 	// Dummy setter to allow JmsRealms being added to Configurations via Frank!Config XSD
+	@Deprecated
 	public void registerJmsRealm(JmsRealm realm) {
 		JmsRealmFactory.getInstance().registerJmsRealm(realm);
```

Comment:
deze regel heeft nooit iets gedaan en veroorzaakt alleen maar verwarring. Dit is een dummy setter, die nooit aangeroepen moet (mag) worden.

Reply:
heb overal redundant code weggehaald